libs/common/src/platform/state/state-definitions.ts

```diff
@@ -150,6 +150,7 @@ export const COLLECTION_DATA = new StateDefinition("collection", "disk", {
   web: "memory",
 });
 export const FOLDER_DISK = new StateDefinition("folder", "disk", { web: "memory" });
+export const FOLDER_MEMORY = new StateDefinition("decryptedFolders", "memory");
```

Comment:
Folders aren't countable and a user could have many of them, for that reason we should use the memory implementation with infinite space on browser. 

```suggestion
export const FOLDER_MEMORY = new StateDefinition("decryptedFolders", "memory", {
  browser: "memory-large-object",
});
```